vrs/vrs_annotation_batch.py

```diff
@@ -71,7 +71,8 @@ def main(args):
 
     # Initialize Hail w/ Query-On-Batch , setting global seed if user decides to downsample
```

Comment:
since there is now on option based on backend_mode, this comment should be updated

Reply:
I think I changed references to QoB some places, but I'll definitely need to go through and make it non-universal